packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -387,6 +413,25 @@ class LocalStoreTester {
     return this;
   }
 
+  toHaveNewerBundle(
+    metadata: BundleMetadata,
+    expected: boolean
+  ): LocalStoreTester {
+    this.promiseChain = this.promiseChain.then(() => {
+      return this.localStore.isNewerBundleLoaded(metadata).then(actual => {
```

Comment:
Name is also inconsistent here. FWIW, `toHaveNewerBundle` matches my name suggestion from above.

Reply:
Agree!